src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml

```diff
@@ -1,28 +1,39 @@
+@using OrchardCore.Admin.Models
 @using OrchardCore.DisplayManagement
 @using OrchardCore.DisplayManagement.ModelBinding
+@using OrchardCore.Entities
+@using OrchardCore.Themes.Services
 @using OrchardCore.Users.Models
 
 @inject IDisplayManager<UserMenu> DisplayManager
 @inject IUpdateModelAccessor UpdateModelAccessor
-
+@inject ThemeTogglerService ThemeTogglerService
+@{
+    var adminSettings = Site.As<AdminSettings>();
+}
 <!DOCTYPE html>
-<html lang="@Orchard.CultureName()">
+<html lang="@Orchard.CultureName()" dir="@Orchard.CultureDir()" data-bs-theme="@await ThemeTogglerService.CurrentTheme()" data-tenant="@ThemeTogglerService.CurrentTenant">
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>@RenderTitleSegments(Site.SiteName, "before")</title>
-    <link type="image/x-icon" rel="shortcut icon" href="~/TheTheme/favicon.ico">
+    <link type="image/x-icon" rel="shortcut icon" href="~/TheTheme/images/favicon.ico">
+
+    <!-- This script can't wait till the footer -->
+    <script asp-name="admin-head" version="1" at="Head"></script>
+    <script asp-name="admin-menu" depends-on="admin-head" version="1" at="Head"></script>
+
+    <style asp-name="bootstrap" version="5" at="Head"></style>
     <script asp-name="bootstrap" version="5" at="Foot"></script>
-    <style asp-name="thetheme-bootstrap-oc" version="1"></style>
+
+    <style asp-name="TheTheme" depends-on="bootstrap" asp-src="~/TheTheme/styles/theme.min.css" debug-src="~/TheTheme/styles/theme.css" at="Foot"></style>
+
     <style asp-name="font-awesome" version="6"></style>
-    <resources type="Meta" />
-    <resources type="HeadLink" />
-    <resources type="Stylesheet" />
-    <resources type="HeadScript" />
+    <resources type="Header" />
     @await RenderSectionAsync("HeadMeta", required: false)
 </head>
 <body dir="@Orchard.CultureDir()">
```

Comment:
Remove `dir="@Orchard.CultureDir()"` from `<body>`, now that you added it on `<html>`.

Reply:
done